components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/FromInstanceTypeDeterminer.scala

```diff
@@ -35,7 +35,7 @@ object FromInstanceTypeDeterminer {
       case other =>
         Typed(other.getClass) match {
           case typedClass: TypedClass =>
-            ToJsonEncoderWithFallback.encodeValue(other) match {
+            ToJsonEncoder.default.encode(other) match {
```

Comment:
Here is a functional change - the result will also be valid fo `DisplayJson` and custom encoders will be used. IMO it will only be better with that approach

Reply:
I remember, that I didn't do that change some time ago because of some failing typing tests after making that modification. But there were a lot of refactors since that time, if tests pass it's great change :+1: